.github/workflows/testing.yml

```diff
@@ -48,9 +48,6 @@ jobs:
       - shell: bash -l {0}
         run: ./build_tools/github/install.sh
         name: 'Install nilearn'
-      - shell: bash -l {0}
-        run: ./build_tools/github/test_docs.sh
-        name: 'Run doctests'
```

Comment:
Hi,
For me to understand, what is the reason ? It fails ?

Reply:
@bthirion Yes doctest fail with Python 3.5 and minimum requirements because of a few scikit-learn related imports.
For example:

```python
Python 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:41:56) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sklearn
>>> sklearn.__version__
'0.19.0'
>>> from sklearn.covariance import GraphicalLassoCV
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'GraphicalLassoCV'
```